EventBrite/Models/Dto/EventBriteEventDto.cs

```diff
@@ -0,0 +1,113 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Etch.OrchardCore.Fields.EventBrite.Models.Dto
+{
+    public class EventBriteEventDto
```

Comment:
This is interesting! You've comprised code conventions to suit the API structure of Eventbrite. This should stick to code conventions we use when writing C# so the code base is consistent. Use `JsonProperty` to handle mapping to the event brite API structure to properties within this class.

Reply:
This is what comes out of the "Paste JSON as Classes" feature within Visual Studio, so blame that 🙈  I'll get them updated using `JsonProperty`